### PR TITLE
feat: [STO-3514]: Security Stage

### PR DESCRIPTION
**Pull request by chris-harness**
##### Summary:

Add a new "Security" Pipeline Stage. Only the minimal boilerplate necessary for the stage to appear in the UI is included in this PR.

##### Jira Links:

https://harness.atlassian.net/browse/STO-3514

##### Screenshots:
Screen Shot 2022-03-21 at 1 57 10 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by tan-nhu — DISMISSED** [review]

**Review by tan-nhu — APPROVED** [review]

File: src/modules/75-ci/pages/pipeline-studio/CIPipelineStudio.tsx [hunk 1]
```diff
@@ -81,6 +82,7 @@ const CIPipelineStudio: React.FC = (): JSX.Element => {
           licenseInformation['CI'] && isCIEnabled,
           licenseInformation['CD'] && isCDEnabled,
           licenseInformation['CF'] && isCFEnabled,
+          isSTOEnabled,
```
Comment (tan-nhu) [inline src/modules/75-ci/pages/pipeline-studio/CIPipelineStudio.tsx:85]: Should this combine with `licenseInformation['CI']` like line 82?

**Review by tan-nhu — APPROVED** [review]

File: src/modules/75-ci/pages/pipeline-studio/CIPipelineStudio.tsx (hunk 1 above)
Comment (chris-harness) [reply src/modules/75-ci/pages/pipeline-studio/CIPipelineStudio.tsx:85]: No, according to Sid, this will be solely gated on Feature Flags until GA

**Review by mtran7 — APPROVED** [review]

**Comment by chris-harness** [comment]
retrigger eslint

**Comment by chris-harness** [comment]
retrigger jest

**Review by mtran7 — APPROVED** [review]

**Review by tan-nhu — APPROVED** [review]

**Pull request merged**